pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -146,18 +148,24 @@ public class NamespaceService implements AutoCloseable {
 
     private final RedirectManager redirectManager;
 
-
+    /** @deprecated by {@link #lookupRedirectsCounter} */
```

Comment:
Should we consider adding an ad-hoc Java annotation here. It would make our life easier when we'll remove these metrics.

Reply:
Yes, something like `@PulsarDeprecatedMetric`. It would easier later on to go and find all of them